lib/ansible/plugins/callback/foreman.py

```diff
@@ -0,0 +1,246 @@
+# (C) 2015, 2016 Daniel Lobato <elobatocs@gmail.com>
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+from datetime import datetime
+from collections import defaultdict
+import json
+import requests
+import time
+
+try:
+    from ansible.plugins.callback import CallbackBase
+    parent_class = CallbackBase
+except ImportError:
+    parent_class = object
```

Comment:
unsure of why this is happening, since you are adding to ship with ansible, callbackbase should always be available